net/shadowsocks-libev/files/ss-rules/chain.uc

```diff
@@ -97,7 +97,7 @@ chain ss_rules_dst_{{ proto }} {
 
 {%   if (proto == "tcp"): %}
 chain ss_rules_forward_{{ proto }} {
-	meta l4proto tcp redirect to :{{ redir_port }};
+	meta l4proto tcp {{ o_tcp_extra }} redirect to :{{ redir_port }};
```

Comment:
chain `ss_rules_local_out` also needs to be filtered.

Reply:
I've checked, it seems to work fine for local out without changing `ss_rules_local_out`.